modules/quotas-requiring-explicit-quota.adoc

```diff
@@ -4,48 +4,131 @@
 
 // NOTE: This is currently not configurable in 4.1, removing from 4.1 docs.
 
-[id="quota-requiring-explicit-quota_{context}"]
-= Requiring explicit quota to consume a resource
+[id="configuring-explicit-resource-quotas_{context}"]
+= Configuring explicit resource quotas
 
-If a resource is not managed by quota, a user has no restriction on the amount
-of resource that can be consumed.  For example, if there is no quota on storage
-related to the gold storage class, the amount of gold storage a project can
-create is unbounded.
+Configure explicit resource quotas in a project request template to apply specific resource quotas in new projects.
 
-For high-cost compute or storage resources, administrators might want to require
-an explicit quota be granted in order to consume a resource.  For example, if a
-project was not explicitly given quota for storage related to the gold storage
-class, users of that project would not be able to create any storage of that
-type.
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+
+* Install the OpenShift CLI (`oc`).
```

Comment:
```
Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
```
This is the language that accepted in the [contribution guide](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#openshift-cli).

Reply:
Thank you for your further review. I had seen your previous suggestion to change this, but following comment https://github.com/openshift/openshift-docs/pull/22882#discussion_r440301564 I used the shorthand. I have now changed it to the longer version as suggested.